Corona-Warn-App/src/test/java/android/util/Log.java

```diff
@@ -0,0 +1,39 @@
+package android.util;
+
+public class Log {
```

Comment:
Do we want to add this class in the repo or did you forget to not commit it?

Reply:
I wanted to keep for testing as it allows unit tests to pass if their use Log.d, seems nice :grin:.